packages/pie-inline-choice/src/main.jsx

```diff
@@ -0,0 +1,61 @@
+import { green, orange, pink } from 'material-ui/colors';
+import { withStyles, withTheme } from 'material-ui/styles';
+import InlineChoice from "./inline-choice.jsx";
+
+import { createMuiTheme, MuiThemeProvider } from 'material-ui/styles';
+import React from 'react';
+import classNames from 'classnames';
+
+const styleSheet = theme => {
```

Comment:
I think we can remove all the styling for now, we can add it later. Alot of these css vars are for components that we aren't going to use in this comp. So you can go depend on `inline-choice` from `index.js` instead of going via `main`

Reply:
Ok Removed. I thought we are gonna use theme.